Packs/Thycotic/Integrations/Thycotic/Thycotic.py

```diff
@@ -465,8 +454,26 @@ def secret_rpc_changepassword_command(client, secret_id: str = '', newPassword:
     )
 
 
+def test_module(client) -> str:
+    if client._is_fetch_credential:
+        if len(client._credential_objects) == 0:
+            return "Enter secret name."
+
+        if client.getCredentials() == "":
+            return "Failed search secret name for sync"
```

Comment:
Why does it fail when there's more than one?

Reply:
Processing only one collection.
Since different secret names can have the same username.
And there will be an error like the last demo.